File: modul 5/soal 5/PRAK405-2210817220034-Wanda Karliyanti.py
i=1
while i<=3:
    a,b = input().split()
    a=int(a)
    b=int(b)
    for j in range (1, a+1):
        k=j
        while k>1:
            print("(%d*%d) + "%(k, b), end="");k-=1
        o=1 ;hasil=(j*b)
        while o<j:
            hasil+=(o*b); o+=1
        print("(%d*%d) = %d"%(k, b, hasil))
    p=1; q=1 ;t=0
    while p<=a:
        t+=q*b ;q+=p+1 ;p+=1
    print("%d" %(t))
    print("\n")
    i+=1

File: modul 5/soal 5/PRAK405-2210817220034-Wanda Karliyanti.c
#include <stdio.h>

int main(){

int a, b, i, j, k, o, p, q, hasil, t;
i=1;
while(i<=3){
scanf("%d %d", &a, &b);

for (j = 1; j <=a; j++){
    for(k=j; k>1; k--){
        printf("(%d*%d) + ", k, b);}
    for(o=1, hasil=o*b; o<j; o++, hasil+=(o*b));{
        printf("(%d*%d) = %d\n",k,b,hasil);}
    for (p=1,q=1,t=0; p<=a; q+=p+1,p++){
        t+=q*b;}
}
    printf("%d",t);
    i+=1;
}
}
